environment_set_up/README.md

```diff
@@ -1,22 +1,32 @@
+The `environment_set_up` directory contains instructional materials and helper scripts to set up your computational environment for running HyTEST workflows and scripts in this repository.
+
+The scripts in this repository are designed to be portable between a cloud environment and on-prem HPC resources. We prioritize work in the cloud because anyone can create an account there to run these workflows, but given that a large number of our users work on the USGS HPC systems, we have also build out some instructional materials to help set up these workflows in those environments as well.
+
+# Cloud Environment
```

Comment:
@rsignell-usgs I asked Gene to review since he has built most of this material, but it might be good to get your eyes on this cloud information I just added in case I'm missing anything important.